src/typings/file.ts

```diff
@@ -46,6 +48,13 @@ const FILE_TYPE_DEFINITIONS: FileTypeDef[] = [
         name: "JSON Lines",
         signature: ["{".charCodeAt(0)],
     },
+    {
+        DecoderFactory: ZstdDecoder,
+        checkIsStructured: () => false,
+        extensions: [".zst", ".zstd"],
```

Comment:
it's a good mindset to brainstorm different file extensions, though in the IANA registry only `.zst` is claimed: https://www.iana.org/assignments/media-types/application/zstd

```suggestion
        extensions: [".zst"],
```